src/problems/sudoku.clj

```diff
@@ -0,0 +1,124 @@
+(ns problems.sudoku)
+;;OH BOY KIDS HERE WE GO
+(defn distinct?
```

Comment:
What the heck? Why in the world would you overwrite `distinct?`. And why would you use the same name (which generates a warning) instead of using a new name?

@HFellows mentioned this earlier today, so I know that you/he did this because the built-in `distinct?` is (according to @HFellows) very slow. I have no idea if that's actually true, but even if it is it makes _no_ sense to be writing your own when you don't even have the system _working_. This is the kind of thing you do after eveything works and you have some _evidence_ that (a) speed is an issue and (b) `distinct?` is responsible for a substantial part of that problem.